src/include/sof/math/trig.h

```diff
@@ -21,17 +22,38 @@
 #define PI_Q4_28      843314857
 #define PI_MUL2_Q4_28     1686629713
 #define CORDIC_31B_TABLE_SIZE		31
-#define CORDIC_SIN_COS_15B_TABLE_SIZE	15
+#define CORDIC_15B_TABLE_SIZE		15
+#define CORDIC_30B_ITABLE_SIZE		30
+#define CORDIC_15B_ITABLE_SIZE		15
 
 typedef enum {
```

Comment:
do we need this anymore?

Reply:
Thanks ,  Added 16 bit  functions and included them.